app/web/features/profile/locales/de.json

```diff
@@ -60,7 +60,7 @@
     "general": "Allgemein",
     "has_housemates": "Hat Mitbewohner",
     "host_drinking": "Trinkt zu Hause",
-    "host_kids": "Hat Kinder",
+    "has_kids": "Hat Kinder",
```

Comment:
I changed these everywhere because it was really confusing me that "Hosts people with kids" was "acceptKids" and "Has kids" was "host_kids".